### PR TITLE
IT Update for 3.7.0

### PR DESCRIPTION
**Pull request by dzpex**

### CODE REVIEW

**Review by ann0see — APPROVED** [review]
Thank you! Just added the official checklist here. There's also another file (.qm) which is empty, but I think that's no problem
- [x] Listed in the Contributors UI https://github.com/jamulussoftware/jamulus/blob/04bba26d365b415f6b1bf646196d5b6c5b3ae8d3/src/util.cpp#L502
- [x] Changes look consistent
- [x] No untranslated strings

File: src/res/translation/translation_it_IT.ts [hunk 1]
```diff
@@ -1572,42 +1572,42 @@
...
         <location filename="../../clientsettingsdlg.cpp" line="177"/>
         <source>Sound card driver settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Settaggi scheda audio</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="178"/>
         <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL let you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
-        <translation type="unfinished"></translation>
+        <translation>Aprirà il software di gestione della scheda audio. I driver danno la possibilità di modificare i valori del buffer delay e in alcuni casi, come con gli ASIO$ALL di scegliere gli input e gli output da usare. Altre informazioni sono disponibili sul sito jamulus.io.</translation>
```
Comment (ann0see) [inline src/res/translation/translation_it_IT.ts:1590]: Ah. sorry: missed this line you typed ASIO$ALL, but I think it should be ASIO4ALL here.

**Review by ann0see — CHANGES_REQUESTED** [review]
Sorry. Missed a typo

**Review by hoffie — CHANGES_REQUESTED** [review]
Looks good, will approve once the comment by @ann0see is addressed. :)

**Comment by ann0see** [comment]
By the way @dzpex you can now open a PR with your changes to the translate3_7_0 branch on the website repo.

File: src/res/translation/translation_it_IT.ts (hunk 1 above)
Comment (dzpex) [reply src/res/translation/translation_it_IT.ts:1590]: yes it's a mistake... i resend the correct file

**Comment by dzpex** [comment]
Thanks for your job!!

**Comment by ann0see** [comment]
Thanks for your job too! BTW: your last commit has an unfinished attribute. Could you please remove it?

**Review by ann0see — APPROVED** [review]
Ok. Thanks!

**Review by hoffie — APPROVED** [review]
Thanks!

**Pull request merged**